README.md

```diff
@@ -65,6 +65,9 @@ export default {
 }
 ```
 
+For addon that supports Fastboot, `ember-fetch` should also be listed as a [peer dependency](https://docs.npmjs.com/files/package.json#peerdependencies). 
```

Comment:
Should `For addon that supports ...` be `For apps that support Fastboot...`?

Reply:
This paragraph is for addon authors, for an addon to support Fastboot it means the app consuming the addon will have no problem to enable Fastboot.

An app can enable Fastboot without `ember-fetch` if none of its addons using `ember-fetch` as well.